src/storage/src/hummock/compactor/mod.rs

```diff
@@ -503,7 +503,7 @@ pub fn start_compactor(compactor_context: Arc<CompactorContext>) -> (JoinHandle<
                                         let (tx, rx) = tokio::sync::oneshot::channel();
                                         let task_id = compact_task.task_id;
                                         shutdown.lock().unwrap().insert(task_id, tx);
-                                        let (compact_task, table_stats) = compactor_runner::compact(context, compact_task, rx).await;
+                                        let (compact_task, table_stats)=  compactor_runner::compact(context.clone(), compact_task, rx).await;
```

Comment:
fmt

Reply:
wired,` cargo fmt i`s invalidated in async move.